src/content/quickstart/gke/_index.md

```diff
@@ -0,0 +1,156 @@
+``` bash
+gcloud container clusters create "cluster-a/b" \
+    --zone "<zone-id>" \
+    --cluster-version "1.17.13-gke.1400" \
+    --username "admin" \
+    --machine-type "g1-small" \
+    --image-type "UBUNTU" \
+    --disk-type "pd-ssd" \
+    --disk-size "15" \
+    --num-nodes "3" \
+    --network "default" \
+    --no-enable-ip-alias
+```
+{{% notice note %}}
+Make sure to use Kubernetes version 1.17 or higher, set by `--cluster-version` (refer to the [GKE docs/ release notes](https://cloud.google.com/kubernetes-engine/docs/release-notes)). Also check for non-overlapping pod and service CIDRs between clusters. If this is not given, use the `--cluster-ipv4-cidr` [option](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create#--cluster-ipv4-cidr). You can check using the command below:
+{{% /notice %}}
+``` bash
+gcloud container clusters describe "cluster-a/b" --zone "<zone-id>" | grep -e clusterIpv4Cidr -e servicesIpv4Cidr
+```
+
+## Prepare clusters for Submariner
+
+Next, the created clusters need some changes in order for Submariner to successfully open the IPSEC tunnel between them.
+
+### Preparation: Node configuration
+
+As of version 0.7 of Submariner (the current one while writing this), Google's native CNI plugin is not directly supported. GKE clusters can be generated with Calico CNI instead, but this was not tested during this demo and therefore could hold surprises aswell.
+
+Configuration is needed for the `eth0` interface of each node on every participating cluster. The used workaround deploys `netshoot` pods onto each node that configure the reverse path filtering. The scripts in [this](https://github.com/sridhargaddam/k8sscripts/tree/main/rp_filter_settings) Github repository need to be executed in all clusters.
+
+``` bash
+wget https://raw.githubusercontent.com/sridhargaddam/k8sscripts/main/rp_filter_settings/update-rp-filter.sh
+wget https://raw.githubusercontent.com/sridhargaddam/k8sscripts/main/rp_filter_settings/configure-rp-filter.sh
+chmod +x update-rp-filter.sh
+chmod +x configure-rp-filter.sh
+echo KUBECONTEXT=cluster-a
+./configure-rp-filter.sh
+echo KUBECONTEXT=cluster-b
+./configure-rp-filter.sh
+```
+
+### Preparation: Firewall configuration
+
+Submariner requires the UDP ports 500, 4500 and 4800 to be open in both directions. Additionally the microservices' traffic needs to flow through the IPSEC tunnel as TCP packets.
+Create firwewall rules on the GCP project that open the UDP ports aswell as all TCP ports with source and destination addresses originating in the participating clusters.
+{{% notice note %}}
+Use the command above to retrieve the `clusterIpv4Cidr` and `servicesIpv4Cidr`.
+{{% /notice %}}
+
+``` bash
+gcloud compute firewall-rules create "allow-tcp-in" --allow=tcp --direction=IN --source-ranges=<pod and service cidrs>
+gcloud compute firewall-rules create "allow-tcp-out" --allow=tcp --direction=OUT --destination-ranges=<pod and service cidrs>
```

Comment:
If we specify the exact CIDRs during creation we can include them here directly , we can add a "#" comment explaining that those are the pod/service cidrs.